plugins/profiles.koplugin/main.lua

```diff
@@ -0,0 +1,83 @@
+local DataStorage = require("datastorage")
+local Event = require("ui/event")
+local FFIUtil = require("ffi/util")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local _ = require("gettext")
+local logger = require("logger")
+local T = FFIUtil.template
+
+local Profiles = WidgetContainer:new{
+    name = "profiles",
+}
+
+local last_profile = 1
+local number_of_profiles = 2
+
+function Profiles:init()
+    logger.info("Profiles:init()")
+    self.ui.menu:registerToMainMenu(self)
+end
+
+function Profiles:addToMainMenu(menu_items)
+    logger.info("Profiles:addToMainMenu")
+    menu_items.profiles = {
```

Comment:
You should also add a `sorting_hint = "tools",`.

PS I haven't tried it yet or thought about where it'd be best to put it.